package.json

```diff
@@ -61,9 +65,7 @@
     "mocha": "^2.2.5",
     "phantomjs": "^1.9.17",
     "react-hot-loader": "^1.2.8",
-    "react-router": "^0.13.3",
-    "react-stub-context": "^0.3.0",
```

Comment:
@Aaike @oliviertassinari I suggested leaving it out because I thought that might fix the build error.

What can we do now to make the build successful?